config/locales/en.yml

```diff
@@ -32,6 +32,7 @@ en:
     support: Support Us
     search: Search
     about: About
+    locale_articles: "English's articles"
```

Comment:
You can delete the wrapping `"` quotes. We don't need those in YAML (unless there's a `:` or something similar in the value).

Reply:
Good to know !